### PR TITLE
changed scaffold and masthead with important tag

### PR DESCRIPTION
**Pull request by whatcoloris**
in hopes of fixing text overflow on 320px iOs

### CODE REVIEW

File: _abouts/masthead.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ permalink: /about/masthead/
 banner: true
 ---
 
-<div class="intro">
+<div class="intro masthead">
   <div class="row">
     <ul class="col s6 m6 l4 collection with-header">
```
Comment (edwardsharp) [inline _abouts/masthead.md:10]: ohey @whatcoloris i think what would work better than this new `!important` css class would be to not have these collections sized `s6` but rather `s12` so a single column takes up the full screen width on small media sizes.

also, the sizes on this have gotten a little weird. and there's now a bunch of dead space on this page. see screenshot 
Screen Shot 2019-04-07 at 2 18 05 PM

i think we should change `col s6 m6 l4 collection with-header` to `col s12 m6 collection with-header` for these collections. and change the `col s12 m12 l9` to `col s12` for the wider collections (Past Editors & so-on) below.

**Review by edwardsharp — CHANGES_REQUESTED** [review]

File: _abouts/masthead.md (hunk 1 above)
Comment (whatcoloris) [reply _abouts/masthead.md:10]: make whatever changes you see fit mah freund!

Comment (edwardsharp) [reply _abouts/masthead.md:10]: okay. but this is your PR 😉

Comment (whatcoloris) [reply _abouts/masthead.md:10]: oh riiiiiiiiiight code of schmonducks

**Pull request closed without merging**